.golangci-new.yml

```diff
@@ -116,11 +123,18 @@ linters:
     - gomoddirectives
     - goprintffuncname
     - gosec
+    - gosmopolitan
     - grouper
     - inamedparam
+    - interfacebloat
+    - intrange
     - ireturn
+    - loggercheck
     - maintidx
     - makezero
+    - mirror
+    - mnd
```

Comment:
It was already disabled as "gomnd", but we need to disable both while it is being transitioned to a new name